assets/js/common/OperationModals/OperationForbiddenModal.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { act, render, screen, within } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import OperationForbiddenModal from './OperationForbiddenModal';
+
+describe('OperationForbiddenModal', () => {
+  it('should show forbidden operation modal', async () => {
+    await act(async () => {
+      render(
+        <OperationForbiddenModal
+          operation="My operation"
+          errors={['error1', 'error2']}
+          isOpen
+        >
+          Some children
+        </OperationForbiddenModal>
+      );
+    });
+
+    expect(screen.getByText('Operation Forbidden')).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        'Unable to run My operation operation. Some of the conditions are not met.'
```

Comment:
operation operation two times ?
I mean sure code wise all perfect,  i was just confused on the first read by it :)